include/kernels/clover_outer_product.cuh

```diff
@@ -30,11 +32,17 @@ namespace quda {
     int displacement;
     bool partitioned[4];
     real coeff;
+    const unsigned int volume_4d_cb;
+    const unsigned int GhostFace_4d_cb;
```

Comment:
Variable name doesn't adhere to our typical convention iirc, but maybe there's precedent for this elsewhere.

Reply:
7acf254ba019063551721afd3969ab77469eb8a0